.github/workflows/freebsd-13.yml

```diff
@@ -0,0 +1,50 @@
+name: freebsd-13
+
+on:
+  push:
+  pull_request:
+  repository_dispatch:
+    types: [backend_automation]
+  workflow_dispatch:
```

Comment:
It seems, it is not updated after PR #6630.

Reply:
I see now, it is updated in a separate commit. I would prefer to make it right from scratch, because this pull request will land after PR #6630 (and given approach with labels is already default for other workflows).

Reply:
fixup'ed in one commit.